data-processing/scripts/run_pipeline.py

```diff
@@ -475,7 +476,10 @@ def run_commands_for_arxiv_ids(
                 # digest for a paper cannot be computed once the paper's data is deleted.
                 pipeline_digest.update(digest_for_paper)
             except S2ApiException as e:
-                logging.error("Retryable Failure processing id: {arxiv_id}", e)
+                logging.exception(f"Retryable Failure processing id: {arxiv_id}")
+                exit(RETRYABLE_FAILURE_RETURN_CODE)
+            except FetchFromArxivException as e:
```

Comment:
could combine into (not necessary, just possible):

```python
except (S2ApiException, FetchFromArxivException) as e:
```

Reply:
Didn't know about this -- handy.